components/switch/homeassistant.rst

```diff
@@ -14,6 +14,11 @@ states and allow **control** via your Home Assistant instance using the :doc:`na
     in Home Assistant, this component still requires you to register the node under Home Assistant. See:
     :ref:`connecting-your-device-to-home-assistant`.
 
+.. warning::
+
+    In order for this component to work, you'll need to allow the device to perform Home Assistant actions by
+    going to the ESPHome integration, clicking 'Configure' on the device and enabling it. 
```

Comment:
This is not always the case. Home Assistant will raise a repair in case it detect this might be needed. I think we should omit this suggested documentation, as it might encourage enabling it by default, which is not a practice we should encourage.

Nevertheless, thanks for being willing to contribute 👍 

../Frenck